src/scripts/sssom_to_robot_template.py

```diff
@@ -0,0 +1,52 @@
+"""Convert SSSOM to ROBOT template."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+from sssom import MappingSetDataFrame
+from sssom.parsers import parse_sssom_table
+
+ROBOT_ROW = {
+    'subject_id': 'ID',
+    'subject_label': '',
+    'object_id': 'A oboInOwl:hasDbXref',
+    'object_label': '',
+    'equivalence': '>A oboInOwl:source',
+    'ORCID': '>A oboInOwl:source SPLIT=|',
```

Comment:
to be more consistent with sssom:

```suggestion
    'author_id': '>A oboInOwl:source SPLIT=|',
```

Reply:
(not needed, if you think its not necessary to be consisent just close the comment)

Reply:
it's good; i'll commit